### DISCUSSION TITLE
Custom SMS provider

### DISCUSSION THREAD

**Discussion by tnsengimana** [supabase/supabase#9136] [Questions] [answered]
Hi - I am trying to use Supabase as the backend of a mobile application in emerging markets, and I'd like to take advantage of Supabase Auth. However, I can not use your supported SMS providers as they don't issue local numbers of the intended markets. I am considering using a custom SMS provider instead. What's your recommendation on using a custom SMS provider?

Thank you.

**Comment by mansueli** [comment, marked as answer]
Hey tnsengimana,

Thanks for creating the discussion here. I believe that you can check this repo:
https://github.com/burggraf/supabase-sms

You'll need to find a transactional SMS provider that supports your region, but tweaking the code there should be straight-forward.
